### PR TITLE
Document --help option in elixir command and include long version of --version

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

File: bin/elixir [hunk 1]
```diff
@@ -15,11 +14,13 @@ if [ $# -eq 0 ] || [ "$1" = "--help" ] || [ "$1" = "-h" ]; then
   --detached                  Starts the Erlang VM detached from console
   --erl SWITCHES              Switches to be passed down to Erlang (*)
   --hidden                    Makes a hidden node
+  --help, -h                  Prints this message and exits
```
Comment (josevalim) [inline bin/elixir]: Alphabetic ordering?

File: bin/elixir.bat [hunk 2]
```diff
@@ -23,11 +22,13 @@ echo   --cookie COOKIE             Sets a cookie for this distributed node
 echo   --detached                  Starts the Erlang VM detached from console
 echo   --erl SWITCHES              Switches to be passed down to Erlang (*)
 echo   --hidden                    Makes a hidden node
+echo   --help, -h                  Prints this message and exits
```
Comment (josevalim) [inline bin/elixir.bat]: Alphabetic ordering?

File: bin/elixir (hunk 1 above)
Comment (eksperimental) [inline bin/elixir]: fixed!

File: bin/elixir.bat (hunk 2 above)
Comment (eksperimental) [inline bin/elixir.bat]: fixed

**Comment by fishcakez** [comment]
Thank you :heart:

**Pull request merged**